Documentation/branch_management.md

```diff
@@ -15,12 +15,12 @@ The `master` branch is our development branch. All new features land here first.
 
 To try new and experimental features, pull `master` and play with it. Note that `master` may not be stable because new features may introduce bugs.
 
-Before the release of the next stable version, feature PRs will be frozen. We will focus on the testing, bug-fix and documentation for one to two weeks.
+Before the release of the next stable version, feature PRs will be frozen. A [release manager](/Documentation/dev-internal/release.md#release-management) will be assigned to major/minor version and will lead the etcd community in test, bug-fix and documentation of the release for one to two weeks.
```

Comment:
Let's fix the markdown paths? https://travis-ci.org/coreos/etcd/jobs/308162491#L834

`[release manager](./dev-internal/release.md#release-management)`?